templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends '@W3CWebsiteTemplates/base.html.twig' %}
+
+{% block main %}
+    <h1>{% block title %}{{ page.title|default('W3C - Error: {{ status_code }} - {{ status_text }}') }}{% endblock %}</h1>
+    <p>The server returned a "{{ status_code }} {{ status_text }}" error.</p>
+
+    {% block stacktrace %}
+        {% if is_granted('ROLE_ADMIN') and exception %}
+            <h2>Error details</h2>
+            <p>Message: <strong>{{ exception.message }}</strong></p>
+            <p>in <em>{{ exception.file }}: {{ exception.line }}</em></p>
+            <p>Trace:</p>
+            <pre>{{ exception.traceAsString }}</pre>
+        {% endif %}
+    {% endblock stacktrace %}
+
+    {% block content %}{% endblock content %}
+    <p>
+        {% set subject = "subject=Error: " ~ (exception ? exception.message : app.request.uri) %}
+        {% set body = "body=Issue description%0A%0A%0A%0A----%0AURL: " ~ app.request.uri ~ "%0ADate: " ~ "now"|date("F jS \\a\\t g:ia") %}
+        {% if exception %}
+            {% set body = body ~ "%0AException at " ~ exception.file ~ ": " ~ exception.line %}
+        {% endif %}
+        If you still have a question, please consult our <a href="/help/">Help page</a> or send
+        us an email to <a href="mailto:sysreq@w3.org?{{ subject }}&{{ body }}">sysreq@w3.org</a>.
```

Comment:
This sentence should also be localized.

Reply:
addressed in https://github.com/w3c/w3c-website-templates-bundle/pull/40/commits/2892c3eccf727bf9f955329c726187c634566c92

Reply:
I'm thinking  that the `<a href="mailto:sysreq@w3.org?{{ subject }}&{{ body }}">sysreq@w3.org</a>.` part also need to be moved to the localization file.

Reply:
addressed in https://github.com/w3c/w3c-website-templates-bundle/pull/40/commits/776d66e897902c8c85388521c3a2561fa634b319

Reply:
Addressed by @deniak in https://github.com/w3c/w3c-website-templates-bundle/pull/40/commits/776d66e897902c8c85388521c3a2561fa634b319